backend/src/main/java/com/happy/friendogly/notification/service/NotificationService.java

```diff
@@ -0,0 +1,10 @@
+package com.happy.friendogly.notification.service;
+
+import java.util.List;
+
+public interface NotificationService {
+
+    void sendNotification(String title, String content, String receiverToken);
+
+    void sendNotification(String title, String content, List<String> receiverToken);
```

Comment:
List는 명시적으로 복수형으로 쓰면 어떨까요??

```suggestion
    void sendNotification(String title, String content, String receiverToken);

    void sendNotification(String title, String content, List<String> receiverTokens);
```

Reply:
List는 명시적으로 복수형으로 쓰면 어떨까요??